lib/config-loader.js

```diff
@@ -138,7 +139,7 @@ module.exports = () => {
   config.ow = userConfig.runtime || {}
   config.ow.apihost = config.ow.apihost || defaultOwApiHost
   config.ow.apiversion = config.ow.apiversion || 'v1'
-  config.ow.package = `${config.app.name}-${config.app.version}`
+  config.ow.package = process.env.WSK_PACKAGE || `${config.app.name}-${config.app.version}`
```

Comment:
@alexkli why do you need the package to be different from the npm package name? 
If possible I would like to avoid this change as it will make the action url less discoverable.

(also I would rename WSK_PACKAGE to AIO_RUNTIME_PACKAGE and run the getModuleName on this as well)

Reply:
if it is because of https://github.com/adobe/wskdebug/issues/49 I would recommend to set the `WSK_PACKAGE` env var here: https://github.com/adobe/aio-app-scripts/blob/master/scripts/dev.js#L216

Reply:
It is needed for Nui. We use custom names.

Reply:
so IIUC you have one app that you deploy into multiple packages into the same namespace?

Reply:
It's more that the naming scheme you chose (`${config.app.name}-${config.app.version}`) does not fit our needs. For instance, we do not want version names in our package names.